bifrost.go

```diff
@@ -13,7 +13,8 @@ import (
 	"github.com/aws/aws-sdk-go-v2/credentials"
 	"github.com/aws/aws-sdk-go-v2/service/s3"
 	"github.com/opensaucerer/bifrost/gcs"
-	bs3 "github.com/opensaucerer/bifrost/s3"
+	"github.com/opensaucerer/bifrost/pinata"
+	"github.com/opensaucerer/bifrost/s3"
```

Comment:
Please check this import rename...it should not be removed. 
The `bs3` rename of the import is necessary to prevent Bifrost's s3 package and the actual s3 SDK from conflicting.